backend/de.metas.material/planning/src/test/java/de/metas/material/planning/ppordercandidate/PPOrderCandidateAdvisedEventCreatorTest.java

```diff
@@ -1,121 +0,0 @@
-/*
- * #%L
- * metasfresh-material-planning
- * %%
- * Copyright (C) 2021 metas GmbH
- * %%
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as
- * published by the Free Software Foundation, either version 2 of the
- * License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public
- * License along with this program. If not, see
- * <http://www.gnu.org/licenses/gpl-2.0.html>.
- * #L%
- */
-
-package de.metas.material.planning.ppordercandidate;
-
-import com.google.common.collect.ImmutableList;
-import de.metas.business.BusinessTestHelper;
-import de.metas.material.planning.event.MaterialRequest;
-import de.metas.quantity.Quantitys;
-import de.metas.uom.UomId;
-import org.adempiere.test.AdempiereTestHelper;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-
-import static org.assertj.core.api.Assertions.*;
-
-class PPOrderCandidateAdvisedEventCreatorTest
```

Comment:
is this test now covered by some new cucumber test?

Reply:
yep => in a way, basically this was testing:
 - Material Dispo creates PP_Order_Candidate taking into account PP_Product_Planning.ManufacturingCapacity; but now there is no such a thing anymore, material dispo will always create just one PP_Order_Candidate for the whole stock shortage and the capacity will come into play when processing those candidates
 
 and the cucumber scenario covers:
 - pp_order_candidates processing logic takes into account Resource.ManufacturingCapacity when creating PP_Order